.github/workflows/tests.yml

```diff
@@ -13,11 +13,14 @@ jobs:
       - uses: actions/checkout@v3
       - uses: software-mansion/setup-scarb@v1
         with:
-          scarb-version: "2.4.0"
+          scarb-version: "2.6.3"
       - run: scarb fmt --check
       - run: scarb build
 
+      - name: Install Universal-Sierra-Compiler
```

Comment:
I thought this is installed as a dependency of SNFoundry, but it seems it's not the case in the CI. I think we have to bump `setup-snfoundry` to [v3](https://github.com/foundry-rs/setup-snfoundry/releases/tag/v3.0.0) to get it. Can you try?

Reply:
oh this sounds likely, I was looking for the wrong thing - a CI for universal-sierra-compiler. I will give this a try.